test/e2e/specs/interactivity/fixtures/index.ts

```diff
@@ -14,12 +14,12 @@ type Fixtures = {
 };
 
 export const test = base.extend< Fixtures >( {
+	// @ts-expect-error: The required type is 'test', but can be 'worker' too. See
+	// https://playwright.dev/docs/test-fixtures#worker-scoped-fixtures
 	interactivityUtils: [
 		async ( { requestUtils }, use ) => {
 			await use( new InteractivityUtils( { requestUtils } ) );
 		},
-		// @ts-ignore: The required type is 'test', but can be 'worker' too. See
-		// https://playwright.dev/docs/test-fixtures#worker-scoped-fixtures
 		{ scope: 'worker' },
```

Comment:
This isn't pretty, but it affects a smaller scope:
```suggestion
		// This is a hack, 'worker' is a valid value but the type is wrong.
		// https://playwright.dev/docs/test-fixtures#worker-scoped-fixtures
		{ scope: 'worker' as 'test' },
```

Reply:
I did think about that but I though that is too hacky but now I realise it wasn't 😄

Reply:
Updated. Thanks